src/cljs/harja/views/hallinta/kojelauta.cljs

```diff
@@ -133,11 +144,11 @@
         (= "vahvistettu" suunnitelman_tila)
         (yleiset/tila-indikaattori "valmis" {:fmt-fn (constantly "Valmis")}))]]))
 
+(def urakoiden-maara-per-sivu 20)
+
 (defn taulukko-paallystysurakat [e!  {:keys [urakat haku-kaynnissa?]}]
   [:div "Tänne tulee lähiaikoina päällystysurakoiden tietoa..."])
 
-(def urakoiden-maara-per-sivu 20)
-
 (defn taulukko-hoitourakat [e! {:keys [urakat haku-kaynnissa?]}]
   [grid/grid
```

Comment:
Ajax-loader ei toimi, jos vaihtelet ELYä. Urakoita haetaan, mutta loaderia ei välttämättä näy jos taulukko ei ole tyhjänä.

Tässä korjaus tähän:

```clojure
(defn taulukko-hoitourakat [e! {:keys [urakat haku-kaynnissa?]}]
  
  (if haku-kaynnissa?
    [:div.ajax-loader-valistys 
     [ajax-loader "Ladataan tietoja"]]

  [grid/grid
   {:otsikko (str "")
    :sivuta urakoiden-maara-per-sivu
    :tyhja "Ei tietoja, tarkistathan valitut suodattimet."
    ;; ...
```

Reply:
Jätin tuon ajax-loaderin sinne :tyhja avaimeen, mielestäni se pitää saada sieltä toimiaan, ja näiden muutosten jälkeen ainakin nyt works on my machine oikein kivasti. Näin ei tarttisi myöskään poistaa gridiä DOM:sta joka kerta kun haetaan tietoa.